src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -149,7 +150,7 @@ export const canShowRRBanner: CanShowFunctionType = async ({
 	}
 
 	const countryCode = await asyncCountryCode;
-	const hasConsentedToArticleCounts = await getArticleCountConsent();
+	const hasConsentedToArticleCounts = await hasCmpConsentForArticleCount();
```

Comment:
This actually needs to call `hasOptedOutOfArticleCount`.
Also, the existing name `hasConsentedToArticleCounts` is not helpful here, as it later gets assigned to a different name when we build the payload:
`hasOptedOutOfArticleCount: !hasConsentedToArticleCounts`

Let's just rename `hasConsentedToArticleCounts` to `hasOptedOutOfArticleCount` in this file, and we can avoid using the `!` operator so much